src/simple/stateless/utils/events.js

```diff
@@ -31,7 +31,7 @@ export default (key, index, first, last, next, prev, options, optType, typedValu
         update.onFiltered(filtered);
       }
       update.onChange(text);
-      document.activeElement.blur();
+      // document.activeElement.blur();
```

Comment:
I can't think of a reason that we would want to call blur() here. This seems to be the thing that is confusing Safari. I would assume that if we just let tab do it's thing that the blur event naturally gets called when we move off of the activeElement onto the next.